developers_chamber/utils.py

```diff
@@ -14,3 +14,16 @@ def call_command(command):
         subprocess.check_call(command, stdout=sys.stdout, shell=isinstance(command, str))
     except subprocess.CalledProcessError:
         raise ClickException('Command returned error')
+
+
+def call_compose_command(command):
```

Comment:
čistě stopne docker compose

první kill -> kontejnery se začnou slušně vypínat
druhý kill - > kontejnery se killují
třetí kill -> zabije se to celé hned